classes/Product.php

```diff
@@ -5168,12 +5168,6 @@ public static function addCustomizationPrice(&$products, &$customized_datas)
                 $product_attribute_id = isset($product_update['id_product_attribute']) ? (int) $product_update['id_product_attribute'] : (int) $product_update['product_attribute_id'];
                 $id_address_delivery = (int) $product_update['id_address_delivery'];
                 $product_quantity = isset($product_update['cart_quantity']) ? (int) $product_update['cart_quantity'] : (int) $product_update['product_quantity'];
-                $price = isset($product_update['price']) ? $product_update['price'] : $product_update['product_price'];
```

Comment:
I think the isset was added for something.
Could you keep the `$price` and `$price_wt` variable instead of using `unit_price_tax_incl` key?

Reply:
In the current release 1.7, product_update come from 3 ways (backtrace):

Products::addCustomizationPrice
Products::addProductCustomizationPrice
Order::getProducts
Order::getProductsDetail > SELECT * FROM `' . _DB_PREFIX_ . 'order_detail` od ...

Products::addCustomizationPrice
Products::addProductCustomizationPrice
OrderInvoice::getProducts
OrderInvoice::getProductsDetail > SELECT * FROM `' . _DB_PREFIX_ . 'order_detail` od ...

Products::addCustomizationPrice
Products::addProductCustomizationPrice
HTMLTemplateOrderSlip::__construct
OrderSlip::getOrdersSlipProducts
Order::getProductsDetail > SELECT * FROM `' . _DB_PREFIX_ . 'order_detail` od ...

So price come from order_details and is already defined.